acceptance/Rakefile

```diff
@@ -1,30 +1,343 @@
-namespace :ci do
+require 'rake/clean'
+require 'pp'
+require 'yaml'
+$LOAD_PATH << File.expand_path(File.join(File.dirname(__FILE__), 'lib'))
+require 'puppet/acceptance/git_utils'
+extend Puppet::Acceptance::GitUtils
 
-  namespace :test do
+ONE_DAY_IN_SECS = 24 * 60 * 60
+REPO_CONFIGS_DIR = "repo-configs"
+CLEAN.include('*.tar', REPO_CONFIGS_DIR, 'merged_options.rb')
 
-    task :aio do
-      if (ENV['SHA']) && (not ENV['SUITE_COMMIT'])
-        ENV['SUITE_COMMIT'] = ENV['SHA']
-      end
-      unless ENV['SUITE_COMMIT'] then
-        fail('SUITE_COMMIT or SHA environment variable must be set to the SHA of the puppet-agent package to test')
+module HarnessOptions
+
+  DEFAULTS = {
+    :type => 'git',
```

Comment:
Should the default be 'aio' ?
Possibly we should strip out anything referring to a git setup; as we don't support such a thing. Or possibly we keep it in, to keep uniform with other projects?